OffchainLabs/arbitrum-token-bridge
# feat: set up PostHog

Initial setup for [Posthog](https://posthog.com/docs/libraries/js) analytics.

This will only capture `page views` for now. Once we make sure it's running we will add all the other events alongside Fathom.

Test steps:
In development mode, posthog will log events in devtools. Only `page views` for now.

You can check the metadata that it's sending to see if there's anything that would be considered an infringement of privacy (given we don't show any banners).

Also open `Application` tab in devtools and check for lack of any posthog cookies there. We should not have any analytics cookies.
